config/routes.rb

```diff
@@ -26,6 +26,13 @@
         end
       end
       resources :room_settings, only: %i[update], param: :friendly_id
+      resources :shared_accesses, only: :destroy, path: '/shared_accesses/room', param: :friendly_id do
+        member do
+          post :create
+          get '/shared_users', to: 'shared_accesses#shared_users'
```

Comment:
Can this not be `index` ?

Reply:
Well we need a endpoint for shared_users, and one for shareable_users.

Reply:
Or we could return both via index? and then filter via frontend?

Reply:
No they're 2 separate endpoints, but `shared_users` is the business definiton of what this is. Sorry not `index`, but instead `show`

Reply:
Since `show` means to return the shared accesses from a specific room

Reply:
You're right.